src/contact.html

```diff
@@ -7,20 +7,32 @@
     <h1 class="title-heading">Contact Us</h2>
 	</div>
 </section>
-<section>
-	<h2> Get in touch </h2>
+<section class="col-lg-3 col-md-7 col-sm-9 col-xs-12 position central">
+	<h2>Get in touch</h2>
 	<form method="POST" action="//formspree.io/help@womenhackfornonprofits.com">
-    <label for="name">Name:</label>
-		<input type="text" name="name" placeholder="E.g. Jon Snow" required>
-    <label for="email">Email Address:</label>
-		<input type="email" name="_replyto" placeholder="E.g. jon.snow@example.com" required>
```

Comment:
`name="_replyto"` is required to make the form submission work, same for other fields that have `name="_smth"`